dvc/dependency/param.py

```diff
@@ -125,9 +125,15 @@ def workspace_status(self):
             elif param not in info:
                 st = "new"
             elif actual[param] != info[param]:
-                st = "modified"
+                if (
```

Comment:
I'm just wondering why the simple `!=` don't work.

```python
In [1]: a = (1,2,3)

In [2]: b = (1,2,3)

In [3]: a == b
Out[3]: True
```

Reply:
It's because we end up converting tuples into lists when we serialize them in dvc.lock. So the actual comparison before was `if [1,2,3] == (1,2,3)` which will evaluate to false.